src/test/java/net/snowflake/ingest/streaming/internal/ParameterProviderTest.java

```diff
@@ -3,6 +3,7 @@
 import java.util.HashMap;
 import java.util.Map;
 import java.util.Properties;
+import net.snowflake.ingest.utils.Constants;
```

Comment:
Should we add an small IT test for GZIP?

Reply:
In which class do you mean?

Reply:
StreamingIngestIT

Reply:
GZIP is the default in this PR.